components/Badge/Badge.tsx

```diff
@@ -30,6 +30,29 @@ interface BadgeProps extends React.HTMLAttributes<HTMLDivElement> {
   variant?: Variant;
 }
 
+export function getBadgeVariant(name: string): Variant {
```

Comment:
The helper method is not being used in component, so placing it here looks a bit strange. Maybe, we can place it in proper file near by?

Reply:
It is a method that should be used together with component, so I would like to keep it in the same file